app/models/location.rb

```diff
@@ -130,6 +130,13 @@ def validate_confirmed
     self.confirmed = [postcode, location_admin_district, location_code, units, type_of_unit, mobility_type].all?(&:present?)
   end
 
+  def linked_local_authorities
+    la = LocalAuthority.find_by(code: location_code)
```

Comment:
This looks confusing to me, but do I remember a comment somewhere else that location_code means something to do with `Location`s but also something else unrelated to do with LAs. Is it just like a unique id for them or something?

Separately, would this be the same as `LocalAuthority.find_by_code(location_code)`?

Reply:
Hm, so I think `lettings_log.location_code` == `location.id`, but `location.code` = `local_authority.location_code` = admin district E code, which is what we're searching by here.

I believe it's the same, yes